sites/feedback/ajax/page-feedback-contact-en.html

```diff
@@ -0,0 +1,71 @@
+<div class="gc-pft row" data-wb-jsonmanager='{
```

Comment:
Suggest to use `id` instead of `class`. we wont have multiple instance of the page feedback for a given page.